source/navitiacommon/navitiacommon/models.py

```diff
@@ -346,11 +346,14 @@ def get_by_name(cls, name):
     @classmethod
     def get_from_id_or_name(cls, id=None, name=None):
         if id:
-            return cls.query.get_or_404(id)
```

Comment:
It's weird that it's not dealing that thing, yes. The "None" case is ok for flask?
Anyway the code is ok for me, just it's weird...